providers/implementations/ciphers/cipher_aes_cbc_hmac_sha256_hw.c

```diff
@@ -733,7 +733,7 @@ static int aesni_cbc_hmac_sha256_set_tls1_aad(void *vctx,
             if (len < AES_BLOCK_SIZE)
                 return 0;
             len -= AES_BLOCK_SIZE;
-            p[aad_len] = len >> 8;
+            p[aad_len - 2] = len >> 8;
```

Comment:
Good find.. Obviously it should match line 727..

Reply:
All the other stitched ciphers look good